website/docs/reference/integrations/datadog.md

```diff
@@ -28,19 +28,57 @@ You can choose to trigger updates for the following events:
 - feature-stale-off
 - feature-environment-enabled
 - feature-environment-disabled
+- feature-variants-updated
+- feature-potentially-stale-on
 
 > *) Deprecated, and will not be used after transition to environments in Unleash v4.3
 
 #### Parameters {#parameters}
 
 Unleash Datadog integration takes the following parameters.
 
-- **Datadog Events URL** - This property is optional. The default url is https://api.datadoghq.com/api/v1/events. Needs to be changed if you are not not on the US1 [Datadog site](https://docs.datadoghq.com/getting_started/site/). Possible alternatives:
+- **Datadog Events URL** - This property is optional. The default URL is https://api.datadoghq.com/api/v1/events. Needs to be changed if you are not not on the US1 [Datadog site](https://docs.datadoghq.com/getting_started/site/). Possible alternatives:
   - EU: https://app.datadoghq.eu/api/v1/events
   - US1: https://app.datadoghq.com/api/v1/events
   - US3: https://us3.datadoghq.com/api/v1/events
   - US1-FED: https://app.ddog-gov.com/api/v1/events
-- **DD API KEY** - This is a required property.
+
+
+- **Datadog API key** - This is a required property. The API key to use to authenticate with Datadog.
+
+- **Datadog Source Type Name** - This property is optional. Sets `source_type_name` parameter to be included in Datadog events.
+
+- **Extra HTTP Headers** - This property is optional. Used to set the additional headers when Unleash communicates with Datadog.
+
+Example:
+
+```json
+{
+  "ISTIO_USER_KEY": "hunter2",
```

Comment:
Does it make sense to send an istio user key to datadog? Maybe we have a more relevant http header to show?

Reply:
Fair. Made this more generic, both in the field placeholder and the example in the docs: https://github.com/Unleash/unleash/pull/4802/commits/afed94f69b2220d3e2101c2865441f9b5dbcf31f